README.md

```diff
@@ -242,6 +233,8 @@ Go to AWS console -> Secrets Manager, then:
 
 #### Upload SSL cert
 
+TODO: finish updating this section; do we still reuse a master cert for all ASF-managed deployments?
+
```

Comment:
No, as of Dec all Tools-managed deployments use a deployment-specific cert provisioned through ACM.

SciDev deployments are still using older certs provisioned through inCommon, including the wildcard *.asf.alaska.edu cert, but per @jhkennedy they might transition to ACM in January.

Reply:
Thanks!